src/Core/Content/ProductExport/Service/ProductExportRenderer.php

```diff
@@ -50,7 +50,7 @@ public function renderHeader(
                 $headerContext->getContext(),
                 $salesChannelContext->getContext()
             ) . \PHP_EOL;
-        } catch (StringTemplateRenderingException $exception) {
+        } catch (AdapterException $exception) {
```

Comment:
same situation we had with the commercial PR :grin: is it okay, if we catch all possible cases of `AdapterException` ?

Reply:
looks not right tbh, the `\Shopware\Core\Framework\Adapter\AdapterException::invalidTemplateSyntax` exception is also thrown in `render` that is not currently caught, i'll check the code again. These exceptions are really not fun :|

Reply:
for such cases the ADR said, we could keep that specific exceptions :grimacing: so we do not need to check the error code :thinking:

Reply:
It's not really working very well, because either people don't know that but more importantly we can't account for the future.